layout/_partial/footer.ejs

```diff
@@ -5,7 +5,7 @@
   <div class="outer">
     <div id="footer-info" class="inner">
       &copy; <%= date(new Date(), 'YYYY') %> <%= config.author || config.title %><br>
-      <%= __('powered_by') %> <a href="http://hexo.io/" target="_blank">Hexo</a>
+      <%= __('powered_by') %> <a href="https://hexo.io/" target="_blank">Hexo</a>
```

Comment:
http://hexo.io returns a 301 redirection to https://hexo.io, so I see no reason to use http link address.